### PR TITLE
[15.0][MIG]website_sale_checkout_skip_payment: Migration to 15.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT36598

please @sergio-teruel @Tardo review this

### CODE REVIEW

**Comment by CarlosRoca13** [comment]
This PR Superseds https://github.com/OCA/e-commerce/pull/632 

Adding the FIX https://github.com/OCA/e-commerce/pull/401 and fixing tests for v15

File: website_sale_checkout_skip_payment/README.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fe--commerce-lightgray.png?logo=github
+    :target: https://github.com/OCA/e-commerce/tree/14.0/website_sale_checkout_skip_payment
+    :alt: OCA/e-commerce
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/e-commerce-14-0/e-commerce-14-0-website_sale_checkout_skip_payment
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/113/14.0
```
Comment (sergio-teruel) [inline website_sale_checkout_skip_payment/README.rst]: Please rebuild this file

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by CarlosRoca13** [comment]
@sergio-teruel done

**Review by sergio-teruel — APPROVED** [review]

**Review by ioans73 — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**